### PR TITLE
Add enum for mouse buttons

### PR DESCRIPTION
**Pull request by goderbauer**
fixes b/26664638

### CODE REVIEW

**Comment by DrMarcII** [comment]
Are there no tests that need updating?

**Comment by goderbauer** [comment]
On my Mac all tests are passing with this change (except for the keyboard chord test, but that one fails on my Mac even on master without the change). So, I guess there were no tests covering this?

File: lib/src/mouse.dart [hunk 1]
```diff
@@ -14,37 +14,51 @@
...
-  static const int middle = 1;
-  static const int right = 2;
+enum MouseButton {
+  // the order of elements in this enum is important, as per
+  // https://w3c.github.io/uievents/#widl-MouseEvent-button
+
+  /// The primary button is usually the left button or the only button on
+  /// single-button devices, used to activate a user interface control or select
+  /// text.
+  primary, // = 0
```
Comment (zoechi) [inline lib/src/mouse.dart]: I don't think it's a good idea to use Dart enums in this case, where it is mandatory that the elements have a specific value. Dart enums aren't built for this purpose.

Comment (goderbauer) [inline lib/src/mouse.dart]: Done.

**Pull request merged**